### PR TITLE
181284020 fonts

### PR DESCRIPTION
**Pull request by alexosugo**
- Updates font from Effra to Latinotype-Brutalista. 
- Font size changes to maintain overall design.
- Updates some viewmodels to get rid of lint errors. 
- Updates gradle.properties rules to improve build time.

### CODE REVIEW

**Review by androidmaven — APPROVED** [review]
Code changes looks good. And app functions as well.

One thing though is that the blinking ticker on editText does not show, This is because the color is the primaryColor instead of white.

Didn't see how the changes in this branch could have affected, so approving this MR. but will make another issue for the above notice.

File: app/src/main/res/layout/add_account_item.xml [hunk 1]
```diff
@@ -1,8 +1,7 @@
...
 <TextView xmlns:android="http://schemas.android.com/apk/res/android"
+        android:id="@+id/account_id"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:fontFamily="font-normal"
-        android:textSize="@dimen/text_16"
         android:textColor="@color/offWhite"
-        android:id="@+id/account_id"/>
\ No newline at end of file
+        android:textSize="@dimen/text_16" />
```
Comment (johnswekenyi) [inline app/src/main/res/layout/add_account_item.xml:7]: Same comment as previous one

File: app/src/main/res/layout/country_item.xml [hunk 2]
```diff
@@ -7,5 +7,4 @@
     android:textSize="@dimen/text_21"
     android:padding="@dimen/margin_13"
     android:textColor="@color/offWhite"
-    android:fontFamily="font-normal"
     android:drawablePadding="@dimen/margin_8"/>
```
Comment (johnswekenyi) [inline app/src/main/res/layout/country_item.xml:10]: New line

File: app/src/main/res/layout/fragment_default_variant.xml [hunk 3]
```diff
@@ -1,64 +1,61 @@
...
-        android:layout_height="match_parent"
-        android:scaleType="centerCrop"
-        android:contentDescription="@string/slide_1_title"
-        android:src="@drawable/stax_slide_2"/>
+            android:id="@+id/slide_Img"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:contentDescription="@string/slide_1_title"
+            android:scaleType="centerCrop"
+            android:src="@drawable/stax_slide_2" />
```
Comment (johnswekenyi) [inline app/src/main/res/layout/fragment_default_variant.xml:13]: Please check your formatting

Comment (johnswekenyi) [inline app/src/main/res/layout/fragment_default_variant.xml:13]: You might want to put it this way `android:src="@drawable/stax_slide_2"/>`

File: app/src/main/res/layout/fragment_default_variant.xml [hunk 4]
```diff
@@ -1,64 +1,61 @@
...
-        android:text="@string/slide_1_title"/>
+            android:id="@+id/slide_text_title"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginHorizontal="@dimen/margin_24"
+            android:layout_marginTop="@dimen/margin_34"
+            android:lineHeight="46dp"
+            android:text="@string/slide_1_title"
+            android:textSize="45sp"
+            app:fontFamily="@font/brutalista_bold" />
```
Comment (johnswekenyi) [inline app/src/main/res/layout/fragment_default_variant.xml:24]: Formatting is off plus remove the space here `app:fontFamily="@font/brutalista_bold"/>`

File: app/src/main/res/layout/fragment_default_variant.xml [hunk 5]
```diff
@@ -1,64 +1,61 @@
...
-        android:text="@string/slide_1_desc"/>
-
+            android:id="@+id/slide_text_desc"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_below="@id/slide_text_title"
+            android:layout_marginHorizontal="@dimen/margin_24"
+            android:layout_marginTop="@dimen/margin_8"
+            android:text="@string/slide_1_desc"
+            android:textSize="@dimen/text_21" />
```
Comment (johnswekenyi) [inline app/src/main/res/layout/fragment_default_variant.xml:34]: Formatting

File: app/src/main/res/layout/recipient_cell.xml [hunk 6]
```diff
@@ -1,8 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <TextView xmlns:android="http://schemas.android.com/apk/res/android"
-	android:layout_width="match_parent"
-	android:layout_height="wrap_content"
-	android:textColor="@color/offWhite"
-	android:textSize="@dimen/text_21"
-	android:fontFamily="font-medium"
-	android:textAlignment="viewEnd" />
```
Comment (johnswekenyi) [inline app/src/main/res/layout/recipient_cell.xml:8]: New line

File: app/src/main/res/layout/recipient_cell.xml [hunk 7]
```diff
@@ -1,8 +1,9 @@
...
\ No newline at end of file
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:textAlignment="viewEnd"
+        android:textColor="@color/offWhite"
+        android:textSize="@dimen/text_21"
+        app:fontFamily="@font/brutalista_medium" />
```
Comment (johnswekenyi) [inline app/src/main/res/layout/recipient_cell.xml:9]: New line too

File: app/src/main/res/layout/transfer_card_edit.xml [hunk 8]
```diff
@@ -1,52 +1,40 @@
...
-<!--        android:layout_height="wrap_content"-->
-<!--        android:hint="@string/request_note_label"-->
-<!--        android:imeOptions="actionDone"-->
-<!--        android:inputType="textCapSentences"-->
-<!--        android:visibility="gone" />-->
+            android:id="@+id/contact_select"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content" />
 
 </com.hover.stax.views.StaxCardView>
```
Comment (johnswekenyi) [inline app/src/main/res/layout/transfer_card_edit.xml:40]: New line

File: app/src/main/res/values/themes.xml [hunk 9]
```diff
@@ -62,8 +66,4 @@
         <item name="snackbarButtonStyle">@style/StaxButton</item>
         <item name="snackbarTextViewStyle">@style/StaxSnackbarText</item>
     </style>
-
-    <style name="SplashTheme" parent="StaxTheme">
-        <item name="android:windowBackground">@drawable/splash_window_background</item>
-    </style>
 </resources>
```
Comment (johnswekenyi) [inline app/src/main/res/values/themes.xml:69]: New line

**Pull request merged**